versioned_docs/version-6.x/function-after-focusing-screen.md

```diff
@@ -47,7 +47,7 @@ In most cases, it's recommended to use the `useFocusEffect` hook instead of addi
 
 ## Triggering an action with the `useFocusEffect` hook
 
-React Navigation provides a [hook](https://reactjs.org/docs/hooks-intro.html) that runs an effect when the screen comes into focus and cleans it up when it goes out of focus. This is useful for cases such as adding event listeners, for fetching data with an API call when a screen becomes focused, or any other action that needs to happen once the screen comes into view.
+React Navigation provides a [hook](use-focus-effect.md) that runs an effect when the screen comes into focus and cleans it up when it goes out of focus. This is useful for cases such as adding event listeners, for fetching data with an API call when a screen becomes focused, or any other action that needs to happen once the screen comes into view.
```

Comment:
I believe that's incorrect reference right now and we rather want to redirect user to something like [this](https://react.dev/reference/react/hooks) 🤔

Reply:
The official React documentation does not have any details about [`useFocusEffect`](https://reactnavigation.org/docs/7.x/use-focus-effect/) and [`useIsFocused`](https://reactnavigation.org/docs/7.x/use-is-focused/), unlike React Navigation, where we have corresponding pages for that.
Should we still change these references?

Reply:
Oh, I see what you did here 😄 I think it's better to redirect to useFocusEffect than to intro to the hooks, great change though